server/src/main/java/org/apache/iotdb/db/engine/cq/CQLogWriter.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.engine.cq;
+
+import org.apache.iotdb.db.conf.IoTDBDescriptor;
+import org.apache.iotdb.db.engine.fileSystem.SystemFileFactory;
+import org.apache.iotdb.db.qp.physical.sys.CreateContinuousQueryPlan;
+import org.apache.iotdb.db.qp.physical.sys.DropContinuousQueryPlan;
+import org.apache.iotdb.db.writelog.io.LogWriter;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.BufferOverflowException;
+import java.nio.ByteBuffer;
+
+public class CQLogWriter implements AutoCloseable {
+
+  private static final String TOO_LARGE_RECORD_EXCEPTION =
+      "Current CQ management operation plan is too large to write into buffer, please increase cqlog_buffer_size.";
+
+  private final ByteBuffer logBuffer;
+  private final LogWriter logWriter;
+
+  public CQLogWriter(String logFilePath) throws IOException {
+    logBuffer = ByteBuffer.allocate(IoTDBDescriptor.getInstance().getConfig().getTlogBufferSize());
+    File logFile = SystemFileFactory.INSTANCE.getFile(logFilePath);
```

Comment:
logFile is not closed in this class.

Reply:
File object doesn't take any file handler and has no close() method, thus there's no need to close it. It is enough that we make sure CQLogWriter and CQLogReader closed after usage.